Firestore/core/test/firebase/firestore/testutil/testutil.cc

```diff
@@ -28,13 +52,161 @@ namespace testutil {
 using model::Document;
 using model::DocumentComparator;
 using model::DocumentSet;
+using model::DocumentState;
 using model::FieldMask;
 using model::FieldPath;
 using model::FieldTransform;
 using model::FieldValue;
 using model::ObjectValue;
 using model::Precondition;
 using model::TransformOperation;
+using nanopb::ByteString;
```

Comment:
I presumed that all the code below was moved with no changes and didn't really review it. Please let me know if I should take a closer look.

Reply:
Correct.